pkg/gotohelm/transpiler.go

```diff
@@ -984,7 +1019,7 @@ func (t *Transpiler) transpileCallExpr(n *ast.CallExpr) Node {
 	// Call to function within the same package. A-Okay. It's transpiled. NB:
 	// This is intentionally after the builtins check to support our bootstrap
 	// package's builtin bindings.
-	if callee.Pkg().Path() == t.Package.PkgPath {
+	if t.namespaceFor(callee.Pkg()) != "" {
```

Comment:
Let's not combine the meaning of namespaces with anything else. This is going to be very difficult to trackdown in the future if we need to make changes as namespace changing doesn't have anything to do with subcharting or dependencies.

Instead, let's introduce a new field on `transpiler` to hold the set of all packages which may be called in this way. `dependencies` could be a good name?

Reply:
When we have idependent packages loaded and our cli args only group do not provide dependencies flag then there is no particular graph direction (one depends on another package) When transpilation starts from root it is fine, but the gotohelm cli will go to another package and it will have the same set of packages included.